frontend/public/extend/devconsole/components/ImportFlowForm/ImportFlowForm.tsx

```diff
@@ -67,6 +67,22 @@ const initialState: State = {
   isBuilderImageDetected: false,
 };
 
+enum UrlErrorType {
```

Comment:
You should define your enum in a way so that you can directly use it to get the error message. You are storing the same information in both key and value here.

Reply:
It should rather be like
```suggestion
enum UrlErrorType {
  RepoNotReachable = 'The git url is not reachable. Please enter the url properly.',
  BranchNotFound = 'The branch could not be found. Please enter the url with the correct branch name.',
}
```
And then it could be as simple as 
```suggestion
const getUrlErrorMessage = (errorType: UrlErrorType) => UrlErrorType[errorType];
```

Reply:
Done